### DISCUSSION TITLE
Rowy compile Error | Firebase: Illegal App name: ' (app/bad-app-name)

### DISCUSSION THREAD

**Discussion by pradeepdurai** [AntlerVC/firetable#999] [Support / Q&A] [answered]
When trying to run the rowy it gives the error : 

_Something went wrong
app/bad-app-name: Firebase: Illegal App name: '   ##(app/bad-app-name)._
![Screenshot from 2022-12-05 10-46-50](https://user-images.githubusercontent.com/55987898/205556313-bafcf4ed-805a-4ba0-b4d7-a93302929f88.png)

Kindly help me to resolve it

**Comment by BeeBombshell** [comment, marked as answer]
Hey @pradeepdurai, does this issue still persist, or was it resolved?

**Reply by BeeBombshell** [reply]
Hey @pradeepdurai, closing this discussion since it has been unattended for a long time. If you're still facing this issue, feel free to create a new one.

**Comment by Usama4745** [comment]
facing same issue
